addons/l10n_ar/models/account_invoice.py

```diff
@@ -0,0 +1,473 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+from dateutil.relativedelta import relativedelta
+import re
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = "account.invoice"
+
+    afip_invoice_concepts = [
```

Comment:
This could be a global variable, out of the class definition
```suggestion
    AFIP_INVOICE_CONCEPTS = [
```